apps/nowcasting-app/components/Toggle.tsx

```diff
@@ -0,0 +1,14 @@
+import { FC } from "react";
```

Comment:
Is it good pratice to put a sentence at the very start of the file, saying roughly what this file does?

Reply:
I've not really seen that in JS/TS world, usually it's a combo of Types and component naming convention to describe what to expect from the component/file